src/SecretManager/Microsoft.DncEng.SecretManager/Commands/SynchronizeCommand.cs

```diff
@@ -63,6 +64,13 @@ public override async Task RunAsync(CancellationToken cancellationToken)
                 }
             }
 
+            Console.OutputEncoding = System.Text.Encoding.UTF8;
+            var problems = new List<(string, string)>();
```

Comment:
nit: unnamed tuples make code less readable
```suggestion
            var problems = new List<(string name, string problem)>();
```